ocs/ocp.py

```diff
@@ -257,11 +257,9 @@ def get_ceph_tools_pod():
         str: The Ceph tools pod name
```

Comment:
As soon as Shylesh implementation of the OCS design will be ready I think we want to return a Pod object here